### PR TITLE
i18n: Add Occitan language

### PR DESCRIPTION
**Pull request by Mejans**
First partially translated version to start with.

### CODE REVIEW

File: po/oc.po [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 3.0.1\n"
+
+#. TRANSLATORS: Don't translate this text
+#: data/com.github.tchx84.Flatseal.desktop.in:4
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:4
+msgid "Flatseal"
+msgstr "Flatseal"
```
Comment (rusty-snake) [inline po/oc.po:25]: > TRANSLATORS: Don't translate this text

Comment (Mejans) [reply po/oc.po:25]: Depends what you call untranslated...it's the same string so it's untranslated, or we should fill in the corresponding msgstr here, thus I am wondering where is there a translation key.

Comment (rusty-snake) [reply po/oc.po:25]: Good question, looks like this is done a bit inconsistent when I look at the other translations.

Comment (Mejans) [reply po/oc.po:25]: ufff, I was asleep when I wrote xD Where=>why

File: po/LINGUAS [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 de
 es
+fr
 hu
 id
 it
 nl
-pt_BR
-sv
+oc
```
Comment (tchx84) [inline po/LINGUAS:12]: if it's not much asking, could you fix the sorting in one commit and add the oc entry together with the second commit? Once the file actually exists.

**Review by tchx84 — CHANGES_REQUESTED** [review]

File: po/LINGUAS (hunk 2 above)
Comment (Mejans) [reply po/LINGUAS:12]: I don't how to do this =/

Comment (rusty-snake) [reply po/LINGUAS:12]: `git rebase -i [--autostash --autosquash] $(git merge-base HEAD master)` and the `edit` the commit. See *SPLITTING COMMITS* in `man 1 git-rebase`.

Comment (Mejans) [reply po/LINGUAS:12]: I did everything via the user interface ^^
If I clone my fork and run this line, will it be enough? I'm really not into git :S

Comment (rusty-snake) [reply po/LINGUAS:12]: > If I clone my fork and run this line, will it be enough?

No.

Comment (Mejans) [reply po/LINGUAS:12]: :(

**Pull request closed without merging**